pages/linux/pdfgrep.md

```diff
@@ -0,0 +1,23 @@
+# pdfgrep
+
+> Search text in PDF files.
+
+- Find lines that match pattern in a PDF:
+
+`pdfgrep {{pattern}} {{file.pdf}}`
+
+- Include file name and page number for each matched line:
+
+`pdfgrep --with-filename --page-number {{pattern}} {{file.pdf}}`
+
+- Find the first 3 lines that begin with foo case insensitive:
```

Comment:
the phrasing "foo case insensitive" sounds a bit odd. How about `Do a case insensitive search that begin with "foo" and return the first 3 lines` ?